### ISSUE TITLE
Is there code for testing on BraTS?

### ISSUE THREAD

**Issue by joangog** [RL4M/PCRLv2#13]
There is code for finetuning (train+val) on BraTS, but there is no testing code so as to evaluate the model on the test set using the dice similarity as done in the paper. Could the authors please provide this?
